src/ergw_aaa_rf.erl

```diff
@@ -292,7 +292,7 @@ handle_aca(['ACA' | #{'Result-Code' := ?'DIAMETER_BASE_RESULT-CODE_SUCCESS'} = A
     {ok, Session, Events, State};
 handle_aca([Answer | #{'Result-Code' := Code}], Session, Events, _Opts, State)
   when Answer =:= 'ACA'; Answer =:= 'answer-message' ->
-    {{fail, Code}, Session, Events, State};
+    {{fail, Code}, Session, Events, State#state{state = stopped}};
 handle_aca({error, _} = Result, Session, Events, _Opts, State) ->
```

Comment:
What is different for Rf?
Nasraq set state to stopped for handle_aca({error, _} 
Rf does not.